src/components/nav/main-nav.js

```diff
@@ -6,7 +6,7 @@ const MainNav = () => {
     <StyledNav>
       <NavList>
         <NavItem>
-          <NavLink href="#1">contact icon</NavLink>
+          <NavLink href="#1">Contact</NavLink>
```

Comment:
we will need to do some icon here with font-awesome... in the meantime this is good.